tools/crm_resource_runtime.c

```diff
@@ -1710,6 +1710,10 @@ cli_resource_execute_from_params(pcmk__output_t *out, const char *rsc_name,
         free(level);
     }
 
+    if (pcmk__str_eq(out->fmt_name, "xml", pcmk__str_none)) {
+        setenv("OCF_OUTPUT_FORMAT", "xml", 1);
+    }
+
```

Comment:
What do you think of doing the setenv() in the XML output constructor?

Reply:
What, like in pcmk__mk_xml_output?  Yeah I think that's reasonable.

Reply:
Or maybe even in the output constructor itself, set OCF_OUTPUT_FORMAT to the format name. Not sure if it's a good idea to go that far but it's easy enough to change if for some reason it isn't